README.md

```diff
@@ -43,15 +43,24 @@ fastify.register(enforceSchema, options);
 
 - **exclude**: Endpoints to exclude by the _routeOptions.path_. Each exclude is an object, with a `url` and optional, `excludeSchemas` array. If the `excludeSchemas` array is not passed, validation for all 3 schemas (`body`, `respone`, `params`) is disabled.
 
-- **excludeOnFalseSchema** - If your controllers aren't returning anything, make sure to set the `response` schema to false. And then, set `excludeOnFalseSchema` to `true`, so that this plugin doesn't return errors for any given controller that has a response schema set to false.
+### **Exclude on schema = false**
```

Comment:
This is a bit confusing, maybe just use the actual property name. Is this actually something we can still set? I don't see it in the example below.